source/includes/examples/tf-example-alerts-replication-lag.rst

```diff
@@ -0,0 +1,29 @@
+.. code-block::
+   :copyable: true
+
+   resource "mongodbatlas_alert_configuration" "test" {
+     project_id = mongodbatlas_project.atlas-project.id
+     event_type = "REPLICATION_OPLOG_WINDOW_RUNNING_OUT"
+     enabled    = true
+
+     notification {
+       type_name     = "GROUP"
+       interval_min  = 5
+       delay_min     = 0
+       sms_enabled   = false
+       email_enabled = true
+       roles         = ["GROUP_CLUSTER_MANAGER"]
+     }
+
+     matcher {
+       field_name = "CLUSTER_NAME"
+       operator   = "EQUALS"
+       value      = "myCluster"
+     }
+
+     metric_threshold_config {
```

Comment:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/13c3385f-ea29-43d9-a217-34c67a526352">

Reply:
Per the API [spec](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration), metric name is not specified for this event type: 
![Screenshot 2024-10-29 at 3 30 28 PM](https://github.com/user-attachments/assets/b9e87627-3795-46f1-b814-0f39d91a767e)

I can do the following instead; can you please let me know if this is valid/correct:

```
resource "mongodbatlas_alert_configuration" "test" {
     project_id = mongodbatlas_project.atlas-project.id
     event_type = "OUTSIDE_METRIC_THRESHOLD"
     enabled    = true
     notification {
       type_name     = "GROUP"
       interval_min  = 5
       delay_min     = 0
       sms_enabled   = false
       email_enabled = true
       roles         = ["GROUP_CLUSTER_MANAGER"]
     }
     matcher {
       field_name = "CLUSTER_NAME"
       operator   = "EQUALS"
       value      = "myCluster"
     }
     metric_threshold_config { 
       metric_name = "OPLOG_SLAVE_LAG_MASTER_TIME"
       operator    = "LESS_THAN"
       threshold   = 1
       units       = "HOURS"
     }
   }
```

Reply:
The new configuration looks good, possibly the semantic meaning is wrong though 🤔 Seems like LESS_THAN 1 HOUR OPLOG_SLAVE_LAG_MASTER_TIME is good and not something to alert about?

Having another look at the original config, you only need to change the name of `metric_threshold_config` -> `threshold_config`:
```
metric_threshold_config { 
       operator    = "LESS_THAN"
       threshold   = 1
       units       = "HOURS"
     }
```
to
```
threshold_config {
    operator  = "LESS_THAN"
    threshold = 1
    units     = "HOURS"
  }
  ```